plugins/mcreator-localization/help/default/rangeditem/projectile.md

```diff
@@ -0,0 +1,4 @@
+This is the projectile that is used by this ranged item.

```

Comment:
This file should probably now be moved to the item folder

Reply:
I already asked Goldorion about which of these files should be kept (check my most recent comment).